app/src/main/java/app/revanced/manager/ui/screen/settings/AboutSettingsScreen.kt

```diff
@@ -130,19 +164,21 @@ fun AboutSettingsScreen(
             }
             FlowRow(
                 maxItemsInEachRow = 2,
-                horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally)
+                horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
+                modifier = Modifier.padding(horizontal = 16.dp)
             ) {
-                filledButton.forEach { (icon, text, onClick) ->
+                preferredSocialButtons.forEach { (icon, text, onClick) ->
                     FilledTonalButton(
-                        onClick = onClick
+                        onClick = onClick,
+                        modifier = Modifier.weight(1f),
                     ) {
                         Row(
                             horizontalArrangement = Arrangement.spacedBy(8.dp),
                             verticalAlignment = Alignment.CenterVertically
                         ) {
                             Icon(
                                 icon,
-                                contentDescription = null,
+                                contentDescription = text,
```

Comment:
This icon should not have a content description because the preferred social buttons already have `Text` elements associated with them unlike the non-preferred socials.

```suggestion
                                contentDescription = null,
```